ui/helpers/constants/common.js

```diff
@@ -9,6 +9,13 @@ export const ERC20 = 'ERC20';
 export const ERC721 = 'ERC721';
 export const ERC1155 = 'ERC1155';
 
+export const TOKEN_STANDARDS = {
+  ERC20,
+  ERC721,
+  ERC1155,
+  NONE: 'NONE',
+};
```

Comment:
Feels like these should eventually live in a module that can be shared across controllers and products. But obviously not a blocker here.